src/Styleguide/Pages/Artist/index.tsx

```diff
@@ -52,7 +53,36 @@ export class Artist extends React.Component {
                           </ContextProvider>
                         </Tab>
                         <Tab name="Shows">
-                          <Shows />
+                          <Sans size="3" weight="medium">
+                            Currently on view
+                          </Sans>
+                          <ContextProvider>
+                            <RelayShowsContent
+                              status="running"
```

Comment:
Each section is the same thing (with `running` shows using the block layout), but with different starting props.